packages/cardpay-reward-programs/cardpay_reward_programs/rules/weighted_usage.py

```diff
@@ -21,7 +21,7 @@ def set_user_defined_parameters(
         self.token = token
         self.duration = duration
 
-    def sql(self, table_query):
+    def sql(self, table_query, aux_table_query=None):
```

Comment:
don't think this is needed?

Reply:
I'll try and either add it everywhere or clear it out of here - it was added I think to match the signature of the function in the base class. With the register refactor this can be standard for all again and take no arguments at all.